### PR TITLE
Added toolbar and headline (Mobile View)

### PR DESCRIPTION
**Pull request by Luiscarlosvd**
- Added a header section with a button for the menu
- Added a headline section in the main with the principal text of the project and social media icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                experience!
+                If you like what you see and have a project you need coded,
+                don't hesitate to contact me.</p>
+            <div class="icons-headline">
+                <img src="img/twitter.png" alt="twitter-icon">
+                <img src="img/linkedin.png" alt="linkedin-icon">
+                <img src="img/medium.png" alt="medium-icon">
+                <img src="img/github.png" alt="github-icon">
+                <img src="img/angellist.png" alt="angellist-icon">
+            </div>
```
Comment (binhussen) [inline index.html:36]: - Kindly, Use a `ul` tag to group the list of icons, and `li` per icon; each icon has a link (`a` tag), this conversion used Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href` :+1: .

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto&display=swap" rel="stylesheet">
+    <title>Luis Valera's Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <button type="button" class="menu-button">
+             <img src="img/Union.png" alt="menu-image">
+        </button>
```
Comment (binhussen) [inline index.html:17]: - Kindly wrap your hamburger button with a `nav` tag for semantics reasons :+1: .

File: style.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
+.header {
```
Comment (binhussen) [inline style.css:1]: - Kindly, by default, some HTML tags have some margin and padding, which leads your background image to have mrgin:8, so please reset it with bellow  :-1:  code 
```
* {
margin: 0;
padding: 0;
}
```

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Luiscarlosvd , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Luiscarlosvd , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**